### PR TITLE
start MELS expanded if it contains a highlighted/permalinked event.

### PR DESCRIPTION
**Pull request by t3chguy**
*For https://github.com/vector-im/riot-web/issues/5038*

### CODE REVIEW

File: src/components/structures/MessagePanel.js [hunk 1]
```diff
@@ -376,15 +381,13 @@ module.exports = React.createClass({
                     eventTiles = null;
                 }
 
-                ret.push(
```
Comment (ara4n) [inline src/components/structures/MessagePanel.js]: slightly surprised that this got reformatted - the original layout was much more legible imo

**Comment by ara4n** [comment]
lgtm - thanks!

**Comment by t3chguy** [comment]
/me was midway through reverting the layout thing

**Comment by t3chguy** [comment]
Why fwiw: 
image

**Comment by ara4n** [comment]
happy to remerge if you push the layout thing if you want. it seems nuts to demand a trailing comma on a foo.push(bar); :/

**Pull request merged**